app/src/main/res/layout/profile_edit_fragment.xml

```diff
@@ -142,7 +143,7 @@
         android:id="@+id/profile_edit_allow_download_container"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@color/color_def_white"
+        android:background="@color/component_color_profile_edit_activity_profile_edit_allow_download_container"
```

Comment:
`component_color_profile_edit_activity_profile_edit_allow_download_container_color`

All component colors should end with suffix `_color`

Reply:
done